vector/src/main/java/im/vector/app/features/pin/PinLocker.kt

```diff
@@ -22,22 +22,29 @@ import androidx.lifecycle.LifecycleObserver
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.OnLifecycleEvent
+import im.vector.app.features.settings.VectorPreferences
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
 import timber.log.Timber
 import javax.inject.Inject
 import javax.inject.Singleton
 
+// 2 minutes, when enabled
 private const val PERIOD_OF_GRACE_IN_MS = 2 * 60 * 1000L
 
+// 1 seconds, to avoid asking for PIN code when switching between Activities
```

Comment:
I don't think this is necessary, we don't lock as long as the app stays in foreground

Reply:
Ok, you are right. I've removed that